api/v1alpha1/power_monitor_internal_types.go

```diff
@@ -39,6 +41,29 @@ type PowerMonitorInternalKeplerConfigSpec struct {
 	// +optional
 	// +listType=atomic
 	AdditionalConfigMaps []ConfigMapRef `json:"additionalConfigMaps,omitempty"`
+
+	// MetricsLevel specifies which metrics levels to export
+	// Valid values are combinations of: node, process, container, vm, pod
+	// +optional
+	MetricsLevel config.Level `json:"metricsLevel,omitempty"`
```

Comment:
Can we rename this to `MetricLevels` to indicate plural. (we should fix kepler.config) and make use of `// +kubebuilder:validation:items:Enum` and make this a string list?

How does the current validation work?